packages/xo-web/src/xo-app/vm/tab-console.js

```diff
@@ -53,6 +57,34 @@ export default class TabConsole extends Component {
     this.setState({ minimalLayout: !this.state.minimalLayout })
   }
 
+  _showFormModal = async () => {
+    const commands = await form({
+      defaultValue: '',
```

Comment:
Should we inject the value of the current state so that the user can also copy a multiline clipboard content coming from the VM?